### PR TITLE
test: ✨ fixed syntax error

### PR DESCRIPTION
**Pull request by Sajal008**
unit test for annoy. errors and bugs with the
syntax fixed

### CODE REVIEW

File: tests/test_annoy_console.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""module contains tests for the Slog class, specifically the annoying method, which is responsible for slog annoying"""
+
+from slogpy.slog import Slog as slog  # noqa: N813,
```
Comment (neel-m) [inline tests/test_annoy_console.py]: nit: extra comma at end of comment

File: tests/test_annoy_console.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Given: a single line annoy message
+
+    When: passed into slog.annoy
+    Then: method will check for annoy message only once and display it.
+    """
+    # Arrange
```
Comment (neel-m) [inline tests/test_annoy_console.py]: nit: why is this here?

File: tests/test_annoy_console.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    # Act
```
Comment (neel-m) [inline tests/test_annoy_console.py]: nit: extra blank line and unnecessary comment

File: tests/test_annoy_console.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+"""module contains tests for the Slog class, specifically the annoying method, which is responsible for slog annoying"""
+
+from slogpy.slog import Slog as slog  # noqa: N813,
+
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Given: a single line annoy message
+
+    When: passed into slog.annoy
+    Then: method will check for annoy message only once and display it.
```
Comment (neel-m) [inline tests/test_annoy_console.py]: This line is not very clear.
What does method mean here?  Is annoy() the method, or is test_annoy_unique_message() the method?
Assuming this test is what you mean by method, the test certainly checks only once, but it doesn't really display it.
Assuming annoy() is what you mean by method, it doesn't check anything, but it should only output the message once.

File: tests/test_annoy_console.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    """
+    # Arrange
+
+    # Act
+    slog.annoy(msg)
```
Comment (neel-m) [inline tests/test_annoy_console.py:15]: Since you are not initializing slog, this test leaves a file behind after it is done.

When run with the other unit tests, it highlights a problem with one of the other ones, since this is the log:

```
2024-06-03 10:56:05,067 - INFO - ANNOYING YOU: This is a annoy test
2024-06-03 10:56:05,078 - INFO - This is a test for the file 
```

Comment (neel-m) [inline tests/test_annoy_console.py:15]: Since you are only calling annoy() once, it is impossible to tell if annoy() if fulfilling its promise to only annoy once.

**Review by neel-m — CHANGES_REQUESTED** [review]

File: tests/test_annoy_console.py (hunk 1 above)
Comment (Sajal008) [reply tests/test_annoy_console.py]: thank you for feedback.

File: tests/test_annoy_console.py (hunk 3 above)
Comment (neel-m) [reply tests/test_annoy_console.py]: unneccessary comment is still there

**Review by neel-m — CHANGES_REQUESTED** [review]

File: tests/test_annoy_console.py (hunk 3 above)
Comment (neel-m) [reply tests/test_annoy_console.py]: this appears to be fixed

**Review by neel-m — APPROVED** [review]
Looks good.
